### DISCUSSION TITLE
Implement MQs like RabbitMQ and Kafka in occlum.

### DISCUSSION THREAD

**Discussion by wydxyreh** [occlum/occlum#1200] [Ideas]
Hi developers, I'm a postgraduate from `Arclab` in `Zhejiang University`. My work is to implement `MQs` in `TEE`, now I'm using `RabbitMQ` in `Occlum`. After discussion with my tutor, I'm going to learning about the source code of `Occlum` and try to modify it. And I notice that you are courious about enabling `Kafka` in `Occlum`. So in my opinion, `Kafka` is similiar to `RabbitMQ` and I'm willing to enable `Kafka` and `RabbitMQ` in `Occlum`, but I have some problem with reading the source code of `Occlum`. Can you give me some guidance and I can help you build those modules?

**Comment by rodrigo-pedro** [comment]
I'm also interested in the implementation of Kafka in Occlum. Keep me updated if you manage to figure it out.
